tabbycat/notifications/utils.py

```diff
@@ -20,7 +20,7 @@
 from options.utils import use_team_code_names
 from participants.models import Person
 from participants.prefetch import populate_win_counts
-from results.result import BaseConsensusDebateResultWithSpeakers, DebateResult, VotingDebateResult
+from results.result import ConsensusDebateResultWithSpeakers, DebateResult, DebateResultByAdjudicatorWithScores
```

Comment:
This should be `ConsensusDebateResultWithScores`, right?

(Will review the rest later, but I noticed this so thought I'd check in on it quickly.)